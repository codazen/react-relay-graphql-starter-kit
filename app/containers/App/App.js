/* @flow */

import React from 'react';

type Props = {
  children: any,
};

export default class App extends React.Component {

  props: Props;

  render() {
    return (
      <div className="container">
        {this.props.children}
      </div>
    );
  }
}
