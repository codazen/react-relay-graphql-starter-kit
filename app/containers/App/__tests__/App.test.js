import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import App from '../App';

global.React = React;

describe('App component', () => {
  it('Renders correctly', () => {
    const app = shallow(<App />);
    expect(shallowToJson(app)).toMatchSnapshot();
  });
});
