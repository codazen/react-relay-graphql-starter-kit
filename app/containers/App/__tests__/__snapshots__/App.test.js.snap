exports[`App component Renders correctly 1`] = `
<div
  className="container" />
`;
